testlib/src/main/java/io/spine/testing/Assertions.java

```diff
@@ -77,12 +76,23 @@ public static IllegalStateException assertIllegalState(Executable e) {
     }
 
     /**
-     * Asserts that running the passed executable causes {@link UnknownTypeException}.
+     * Asserts that running the passed executable causes {@code io.spine.type.UnknownTypeException}.
+     *
+     * @implNote The exception class belongs to the {@code io.spine.base:spine-base} artifact.
+     *           In order to avoid the dependency on {@code spine-base} from {@code spine-testlib}
+     *           We obtain the class of the exception via its name.
      */
     @CanIgnoreReturnValue
-    public static UnknownTypeException assertUnknownType(Executable e) {
+    public static Throwable assertUnknownType(Executable e) {
```

Comment:
In SpineEventEngine organisation, there's just [as much usage scenarios](https://github.com/search?q=org%3ASpineEventEngine%20assertUnknownType&type=code), and all of them are in `base`. 

I suggest we just leave this method in `base`, perhaps, as a Kotlin extension.